server/routes.js

```diff
@@ -9,6 +9,7 @@
     require('./routes/files')(router);
     require('./routes/error')(router);
     require('./routes/upload')(router);
+    require('./routes/downloadRouter')(router);
```

Comment:
I agree with you.
Is it common to name routes without 'Router' suffix prefix ? I got confused by different file with same name. 
For example:
* _client/app/directives/upload.js_
* _client/app/services/Upload.js_
* _server/routes/upload.js_